### DISCUSSION TITLE
Pause and exit the game immediately after the game starts, the pause sound effect will not stop immediately

### DISCUSSION THREAD

**Discussion by Jan-Arbus** [ppy/osu#20626] [Q&A]
https://user-images.githubusercontent.com/74862110/194707908-6a3e935e-2cf0-4ca9-ab3d-3b66df354c75.mp4

**Comment by Jan-Arbus** [comment]
This bug does not appear in Beatmaps with Storyboards and Videos

**Comment by smoogipoo** [comment]
I can't seem to reproduce this, is anyone else able to?

**Reply by peppy** [reply]
I believe this is going to be either due to the audio thread running slow (taking time to remove channels) or GC not running immediately (and leaving samples playing until it runs). Haven't confirmed which, but either will explain why we can't repro (I tried locally and couldn't either).
